purchase_isolated_quotation/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Copyright (C) 2012 Ecosoft (<http://www.ecosoft.co.th>)
+#
```

Comment:
change to short headers following [OCA template](https://github.com/OCA/maintainer-tools/tree/master/template/module)